Validation.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhoneBookApp
+{
+    internal class Validation
+    {
+        public static bool IsNameValid(string? stringInput)
+        {
+            if (String.IsNullOrEmpty(stringInput) || stringInput.Length > 50)
```

Comment:
You can simplify String to string, but this is personal preference (I always lean toward more simplicity) 
![image](https://user-images.githubusercontent.com/108711476/202898697-749eac8f-6b13-4149-8c6c-e415fc33c514.png)